src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -110,9 +94,9 @@ tests:
               - name: "AttestationSignature"
                 saveAs: attestationSignature
 
-      # verification: "Step 5: Verify that the DUT responds with the CSRResponse command."
+      # verification: "Verify that the DUT responds with the CSRResponse command."
     - label:
-          "Step 6.1: Read the commissioner root certificate from TH2's fabric.
+          "Step 4: Read the commissioner root certificate from TH2's fabric.
```

Comment:
```suggestion
          "Step 5.1: Retrieve the the commissioner root certificate from TH2
```

Reply:
Could you please move this down to step 5 so it's clear why this is being done?

Reply:
Fixed: I moved the step to 5.1